documentation/src/main/asciidoc/reference/mapper-orm-indexing-massindexer.asciidoc

```diff
@@ -77,6 +77,30 @@ include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/indexing/Hibe
 <2> Start the mass indexing process for the selected types and return when it is over.
 ====
 
+You can also select a subset of target entities to be reindexed.
```

Comment:
This section is already quite long... maybe move your additions to a dedicated section, just above "`MassIndexer` parameters"?

E.g.:

```
[[mapper-orm-indexing-massindexer-conditional]]
== Conditional reindexing

[...]
```